File: app/src/main/java/com/artmurka/artmurkaapp/Views/Fragments/Interfaces/IItemListFragment.java
package com.artmurka.artmurkaapp.Views.Fragments.Interfaces;

import com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemList.GoodsProperties;

import java.util.ArrayList;

public interface IItemListFragment {
    void showItemList(ArrayList<GoodsProperties> goodsProperties); //какой то эррейлист
    void showError(String error);
    void setTitle(String title);
}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/IItemListFragment.kt
package com.artmurka.artmurkaapp.android.views.fragments.interfaces

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist.GoodsProperties
import com.artmurka.artmurkaapp.presenter.PresenterView

import java.util.ArrayList

interface IItemListFragment  : PresenterView {
    fun showItemList(goodsProperties: ArrayList<GoodsProperties>)  //какой то эррейлист
    fun showError(error: String)
    fun setTitle(title: String)
    fun stopProgressBar()
}
